ontology/yaml/resources/states/states.yaml

```diff
@@ -90,3 +90,14 @@ SOFTSTART: "Mode of operation of rectifier where the input power supply is gradu
 NORMAL: "Function, state or flow which is an ideal or expected condition."
 REVERSED: "Function, state or flow which is opposed to the expected or normal condition."
 
+# For elevator 
```

Comment:
Descriptions need to be different from each other. Make more general so they can apply to non-elevators in the future if needed.

Reply:
changed, hopefully this is better